src/Components/daterange/MonthRange.svelte

```diff
@@ -0,0 +1,77 @@
+<script>
```

Comment:
I believe this could be mixed into https://github.com/6eDesign/svelte-calendar/blob/master/src/Components/Month.svelte based on if selectedEnd / visibleNextMonth is set.  Or even just have a `range = true/false` bool.